include/crm/common/ipc_internal.h

```diff
@@ -111,30 +111,40 @@ struct pcmk__remote_s {
     mainloop_io_t *source;
 
     /* CIB-only */
-    bool authenticated;
     char *token;
 
     /* TLS only */
 #  ifdef HAVE_GNUTLS_GNUTLS_H
     gnutls_session_t *tls_session;
-    bool tls_handshake_complete;
 #  endif
 };
 
 enum pcmk__client_flags {
     // Lower 32 bits are reserved for server (not library) use
 
     // Next 8 bits are reserved for client type (sort of a cheap enum)
-    pcmk__client_ipc        = (UINT64_C(1) << 32), // Client uses plain IPC
-    pcmk__client_tcp        = (UINT64_C(1) << 33), // Client uses TCP connection
+    pcmk__client_ipc                    =
+        (UINT64_C(1) << 32), // Client uses plain IPC
+    pcmk__client_tcp                    =
+        (UINT64_C(1) << 33), // Client uses TCP connection
 #  ifdef HAVE_GNUTLS_GNUTLS_H
-    pcmk__client_tls        = (UINT64_C(1) << 34), // Client uses TCP with TLS
+    pcmk__client_tls                    =
+        (UINT64_C(1) << 34), // Client uses TCP with TLS
 #  endif
 
     // The rest are client attributes
-    pcmk__client_proxied    = (UINT64_C(1) << 40), // Client IPC is proxied
-    pcmk__client_privileged = (UINT64_C(1) << 41), // root or cluster user
-    pcmk__client_to_proxy   = (UINT64_C(1) << 42), // Local client to be proxied
+    pcmk__client_proxied                =
+        (UINT64_C(1) << 40), // Client IPC is proxied
+    pcmk__client_privileged             =
+        (UINT64_C(1) << 41), // root or cluster user
+    pcmk__client_to_proxy               =
+        (UINT64_C(1) << 42), // Local client to be proxied
+    pcmk__client_authenticated          =
+        (UINT64_C(1) << 43), // Client is authenticated (CIB only)
+#  ifdef HAVE_GNUTLS_GNUTLS_H
+    pcmk__client_tls_handshake_complete =
```

Comment:
Any objections to this indentation? Line-wrapped to stay under 80 characters and keep the `=` aligned, but this may warrant an exception. That, or a shorter (and less descriptive) name than `pcmk__client_tls_handshake_complete`.